README.md

```diff
@@ -1,2 +1,52 @@
-# synternet-chain-releases
-This repository contains all the published binaries of the Synternet Chain
+# Synternet Chain releases
+
+This repository hosts Synternet Chain releases of binary artifacts.
+
+It also contains a guide for running Validator node.
```

Comment:
This readme mentions guide for running a validator node, but it is nowhere to be found in this repo.

Reply:
removed this line for the meantime, should be readded later when the guide is ready on our docs